keyboards/handwired/lagrange/config.h

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 Dimitris Papavasiliou <dpapavas@protonmail.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x2718
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Dimitris Papavasiliou
+#define PRODUCT         Lagrange
+#define DESCRIPTION     An ergonomic, split keyboard with thumb clusters.
+
+#define EE_HANDS
+#define SPLIT_USB_DETECT
```

Comment:
Not a huge fan of having this defined by default.  

Is this needed due to design issues?

Reply:
If you're referring to `SPLIT_USB_DETECT`, then according to the documentation, it's required for AVR and I don't know of any other method to determine which side is master and which slave.

If you mean `EE_HANDS`, it is indeed not required here and I missed it when considering what to move to my personal keymap.  Do you think I should remove it altogether and simply let the default configuration be `MASTER_LEFT`?

Reply:
yes, for the `SPLIT_USB_DETECT`.  And no, it's not required for AVR.  In fact, it's not needed for most AVR boards, but it is needed for a number of ChibiOS boards. 

The normal method is by checking VBUS, since that gets power from USB. If it's present, it's the master, because ... usb.  Otherwise, it's not.  

However, the Elite C rev3 had a defect in it (wrong schottky diode) that broke vbus detection, which means those need this. Also, it's enabled by default for ARM boards.

Reply:
Ah yes.  Please excuse my sloppy reading of the documentation; it says it's required for "for AVR Teensy boards".  Well, my board shares the same design flaw (a deliberate choice, to avoid complications on my first PCB design), in that the +5V from the master side is not kept from reaching the `VBus` pin.  I should have led with that...

Reply:
No worries.  But yeah, if you don't have VBUS properly configured, then yes, you would need the split usb detect.  

I just wanted to make sure that it wasn't being unnecessarily added, due to how it initializes. VBUS is ... better.  And not difficult to add, apparently. (I say, recently adding to the docs how to add vbus detection to teensy's)